src/internal/packager/helm/images.go

```diff
@@ -0,0 +1,85 @@
+package helm
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/defenseunicorns/zarf/src/config"
+	"github.com/defenseunicorns/zarf/src/pkg/utils"
+	"github.com/defenseunicorns/zarf/src/types"
+	"github.com/goccy/go-yaml"
+	"helm.sh/helm/v3/pkg/chart/loader"
+)
+
+// ChartImages captures the structure of the helm.sh/images annotation within the Helm chart.
+type ChartImages []struct {
+	// Name of the image.
+	Name string `yaml:"name"`
+	// Image with tag.
+	Image string `yaml:"image"`
+	// Condition specifies the values to determine if the image is included or not.
+	Condition string `yaml:"condition"`
+	// Dependency is the subchart that contains the image, if empty its the parent chart.
+	Dependency string `yaml:"dependency"`
+}
+
+// FindImagesForChartRepo iterates over a Zarf.yaml and attempts to parse any images.
+func FindImagesForChartRepo(repo, path string) (images []string, err error) {
+	matches := strings.Split(repo, "@")
+	if len(matches) < 2 {
+		return images, fmt.Errorf("cannot convert git repo %s to helm chart without a version tag", repo)
+	}
+
+	// Trim the first char to match how the packager expects it, this is messy,need to clean up better
+	repoHelmChartPath := strings.TrimPrefix(path, "/")
+
+	// If a repo helm chart path is specified.
+	component := types.ZarfComponent{
+		Charts: []types.ZarfChart{{
+			Name:    repo,
+			URL:     matches[0],
+			Version: matches[1],
+			GitPath: repoHelmChartPath,
+		}},
+	}
+
+	tmpDir, err := utils.MakeTempDir(config.CommonOptions.CachePath)
```

Comment:
:laughing: :laughing: :laughing: :laughing: :laughing: :laughing: :laughing: :laughing:

Reply:
```suggestion
	tmpDir, err := utils.MakeTempDir(config.GetAbsCachePath())
```

Reply:
Actually DownloadChartFromGit should probably be refactored to support this more DRYly